src/pages/settings/panes/MyBots.tsx

```diff
@@ -352,7 +352,6 @@ function BotCard({ bot, onDelete, onUpdate }: Props) {
                     onClick={() => writeClipboard(bot.token)}
                     description={
                         <>
-                            {"••••••••••••••••••••••••••••••••••••"}{" "}
```

Comment:
What should be used instead?

Reply:
It doesn't really need to be there, but I could probably replace the "Reveal" button text to "Click to Reveal token" if that makes it more obvious.